src/components/Splash.vue

```diff
@@ -63,6 +63,7 @@
         :max-questions-allowed-to-attempt="maxQuestionsAllowedToAttempt"
         :quiz-time-limit="quizTimeLimit"
         :questionSets = "questionSets"
+        class="xl:mr-20 xl:ml-10 lg:mr-24 lg:ml-24 md:mr-48 md:ml-48 sm:mr-60 sm:ml-60"
```

Comment:
this is weird. just check splash screen if its looking good

Reply:
i see that you shifted instructionpage ka class to entire splash, is that right?

Reply:
Yes shifted this class to splash as it was only required in splash screen and not in questionPalette.